app/io/pathfinder/controllers/CommodityController.java

```diff
@@ -0,0 +1,107 @@
+package io.pathfinder.controllers;
+
+import com.avaje.ebean.Ebean;
+import com.avaje.ebean.text.json.JsonContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import play.mvc.*;
+import io.pathfinder.models.Commodity;
+
+import java.util.Iterator;
+
+import static play.libs.Json.*;
```

Comment:
Please don't use wildcard imports, it makes it harder for other people to figure out what the code is doing.